### PR TITLE
adapt more links

### PR DESCRIPTION
**Pull request by cdietrich**

### CODE REVIEW

File: org.eclipse.xtend.maven.parent/pom.xml [hunk 1]
```diff
@@ -18,9 +18,9 @@
 	<description>Xtend is a little language that compiles into idiomatic Java source code. You can use any existing Java library seamlessly from Xtend (and vice-versa). The compiled output is readable and pretty-printed, and tends to run as fast or faster than the equivalent handwritten Java code. It's the CoffeeScript for Java.</description>
 
 	<scm>
-		<connection>scm:git:https://github.com/eclipse/xtext-xtend.git</connection>
-		<developerConnection>scm:git:https://github.com/eclipse/xtext-xtend.git</developerConnection>
-		<url>https://github.com/eclipse/xtext-xtend.git</url>
+		<connection>scm:git:https://github.com/eclipse-xtext/xtext.git</connection>
```
Comment (LorenzoBettini) [inline org.eclipse.xtend.maven.parent/pom.xml:21]: These 3 elements are not required: they are inherited from the parent pom.

**Review by LorenzoBettini — CHANGES_REQUESTED** [review]

File: org.eclipse.xtend.maven.parent/pom.xml (hunk 1 above)
Comment (LorenzoBettini) [reply org.eclipse.xtend.maven.parent/pom.xml:21]: I mean the 3 XML tags you modified

Comment (cdietrich) [reply org.eclipse.xtend.maven.parent/pom.xml:21]: Hmmmhave no longer pc Access can you adjust? We did we have 3

Comment (LorenzoBettini) [reply org.eclipse.xtend.maven.parent/pom.xml:21]: I can do that tomorrow, but I don't understand your answer

Comment (LorenzoBettini) [reply org.eclipse.xtend.maven.parent/pom.xml:21]: In the dev bom is ok: that doesn't inherit from the parent pom.

**Comment by LorenzoBettini** [comment]
@cdietrich I made the changes, removing the XML sections and I also fixed the URL of the Xtend website

**Review by LorenzoBettini — APPROVED** [review]
